systemtest/src/test/java/io/strimzi/systemtest/kafka/KafkaST.java

```diff
@@ -736,7 +736,7 @@ void testRemoveUserOperatorFromEntityOperator(ExtensionContext extensionContext)
         });
 
         timeMeasuringSystem.stopOperation(operationId, extensionContext.getRequiredTestClass().getName(), extensionContext.getDisplayName());
-        assertNoCoErrorsLogged(NAMESPACE, timeMeasuringSystem.getDurationInSeconds(extensionContext.getRequiredTestClass().getName(), extensionContext.getDisplayName(), operationId));
+        assertNoCoErrorsLogged(timeMeasuringSystem.getDurationInSeconds(extensionContext.getRequiredTestClass().getName(), extensionContext.getDisplayName(), operationId));
```

Comment:
I think this might be a good place for issues. In case that CO is in a different namespace than current client uses, it will probably return an empty log and the check will always pass.

Reply:
Yes, that's possible as namepsace was not used at all..

Reply:
Is it possible to use the `namespace` in the `assertNoCoErrorsLogged`? As @Frawless mentioned, it can be good place for issues. (this applies to all other changed lines)

Reply:
I added one function back (which uses default namespace from kubeClient()) and updated other to use specifically called NS. 
This should provide versatility for both cases. 
No tests were updated/reverted back, as they used explicitely Classwide namespace, so it should be good as is.